File: ClientPhp/Model/Nota.php
<?php

class Nota {
    private $nID;
    private $mark;
    private $descriere;
    private $sID;
    private $mID;

    function __construct($nID, $mark, $descriere, $sID, $mID) {
        $this->nID = $nID;
        $this->mark = $mark;
        $this->descriere = $descriere;
        $this->sID = $sID;
        $this->mID = $mID;
    }

    function getnID() {
        return $this->nID;
    }

    function getMark() {
        return $this->mark;
    }
    
    function getDescriere() {
        return $this->descriere;
    }

    function getsID() {
        return $this->sID;
    }

    function getmID() {
        return $this->mID;
    }

    function setnID($nID) {
        $this->nID = $nID;
    }

    function setMark($mark) {
        $this->mark = $mark;
    }

    function setDescriere($descriere) {
        $this->descriere = $descriere;
    }
    
    function setsID($sID) {
        $this->sID = $sID;
    }

    function setmID($mID) {
        $this->mID = $mID;
    }
}

?>

File: ServerJava/src/main/java/ServerJava/Model/Nota.java
package ServerJava.Model;

public class Nota {

    private int nID;
    private int mark;
    private String descriere;
    private int sID;
    private int mID;


    public Nota(int mark, String descriere, int sID, int mID) {
        this.mark = mark;
        this.descriere = descriere;
        this.sID = sID;
        this.mID = mID;
    }

    public Nota() {}

    public int getnID() {
        return nID;
    }

    public void setnID(int nID) {
        this.nID = nID;
    }

    public int getMark() {
        return mark;
    }

    public void setMark(int mark) {
        this.mark = mark;
    }

    public String getDescriere() {
        return descriere;
    }

    public void setDescriere(String descriere) {
        this.descriere = descriere;
    }

    public int getsID() {
        return sID;
    }

    public void setsID(int sID) {
        this.sID = sID;
    }

    public int getmID() {
        return mID;
    }

    public void setmID(int mID) {
        this.mID = mID;
    }
}

